pyproject.toml

```diff
@@ -126,27 +126,16 @@ documentation = "https://docs.saleor.io/"
   fakeredis = "^1.8"
 
 [tool.black]
-target_version = [ "py35", "py36", "py37", "py38" ]
+target_version = [ "py39" ]
```

Comment:
`black` [22.6.0](https://github.com/psf/black/releases/tag/22.6.0) adds support for [py11](https://github.com/psf/black/blob/aafc21aa77f5c4d2ebcb833aa60faba6c2138b94/src/black/mode.py#L125), any reason for not using Python 3.11 mode?

Reply:
I did not want to reformat any files in the process.